index.html

```diff
@@ -46,9 +46,9 @@
 					<svg xmlns="http://www.w3.org/2000/svg" width="20" height="28"><path d="M19.377 28H0V9.656l6.189-6.168A4.924 4.924 0 0 0 7.76 4.547a4.91 4.91 0 0 0 1.923.388 4.91 4.91 0 0 0 1.923-.388 4.924 4.924 0 0 0 1.571-1.059l6.184 6.146V28zM2.655 11.231l-1.371 1.371 4.562 4.561 3.875-3.875 3.876 3.875 4.561-4.561-1.37-1.371-3.191 3.191-3.876-3.875-3.875 3.875-3.19-3.191zm7.034-8.227a2.982 2.982 0 0 1-1.169-.236 3 3 0 0 1-.955-.644L9.692.004l2.13 2.117a3 3 0 0 1-.955.644A2.982 2.982 0 0 1 9.688 3z"/></svg>
 				</a>
 				<span class="spacer"></span>
-				<span class="spacer"></span>
```

Comment:
Unfortunatly the "spacer" class gets removed on mobile phones (to adjust the layout). This could just be a <span>/<span> if you really perfer just two items.